### DISCUSSION TITLE
cheap vps suggestion

### DISCUSSION THREAD

**Discussion by isinteger** [Tkd-Alex/Twitch-Channel-Points-Miner-v2#372] [Q&A]
Can anyone recommend a cheap vps on which will be only this script?

**Comment by ColinShark** [comment]
Extremely late answer, but I'm using uberspace (https://uberspace.de/en).

Shared hosting, so no `sudo` or anything, but its PWYW with a minimum of 1€/month. Current and next full month are for free, then it's Pre-Paid, no subscription.

**Comment by vsevolodmelnyk** [comment]
use heroku, it`s free

**Reply by Scorpiorodrigo** [reply]
I used this script so the script worked prorpely https://github.com/michaelkryukov/heroku-python-script
Run the twitch-miner and authenticate before git push

**Comment by postyy2** [comment]
uberspace is the best
